new-website/utils/tutorials/fetch_tutorials.py

```diff
@@ -12,83 +12,102 @@
 import os
```

Comment:
Let's just update the README to explain that we use the github API and no longer use scraping

Reply:
Updated the corresponding docs.